tests/vklayertests_ray_tracing_pipeline.cpp

```diff
@@ -1199,6 +1216,92 @@ TEST_F(VkLayerTest, RayTracingTestWrongPipelineType) {
     m_errorMonitor->VerifyFound();
 }
 
+TEST_F(VkLayerTest, RayTracingPipelineLibraryGroupHandlesEXT) {
+    TEST_DESCRIPTION("Validate VK_EXT_pipeline_library_group_handles");
+    SetTargetApiVersion(VK_API_VERSION_1_2);
+    AddRequiredExtensions(VK_EXT_PIPELINE_LIBRARY_GROUP_HANDLES_EXTENSION_NAME);
+    if (!InitFrameworkForRayTracingTest(this, true)) {
+        GTEST_SKIP() << "unable to init ray tracing test";
+    }
+    if (DeviceValidationVersion() < VK_API_VERSION_1_2) {
+        GTEST_SKIP() << "At least Vulkan version 1.2 is required";
+    }
+    auto ray_tracing_features = LvlInitStruct<VkPhysicalDeviceRayTracingPipelineFeaturesKHR>();
+    auto pipeline_library_group_handles_features =
+        LvlInitStruct<VkPhysicalDevicePipelineLibraryGroupHandlesFeaturesEXT>(&ray_tracing_features);
+    auto features2 = GetPhysicalDeviceFeatures2(ray_tracing_features);
+
+    if (!ray_tracing_features.rayTracingPipeline) {
+        GTEST_SKIP() << "Feature rayTracing is not supported.";
+    }
+    if (!pipeline_library_group_handles_features.pipelineLibraryGroupHandles) {
+        GTEST_SKIP() << "Feature pipelineLibraryGroupHandles is not supported.";
+    }
+    if (!ray_tracing_features.rayTracingPipelineShaderGroupHandleCaptureReplay) {
+        GTEST_SKIP() << "rayTracingShaderGroupHandleCaptureReplay not enabled";
+    }
+    pipeline_library_group_handles_features.pipelineLibraryGroupHandles = false;
+    ASSERT_NO_FATAL_FAILURE(InitState(nullptr, &features2));
+    const VkPipelineLayoutObj pipeline_layout(m_device, {});
+
+    const char *empty_shader = R"glsl(
+        #version 460
+        #extension GL_EXT_ray_tracing : require
+        void main() {}
+    )glsl";
+    VkShaderObj chit_shader(this, empty_shader, VK_SHADER_STAGE_CLOSEST_HIT_BIT_KHR, SPV_ENV_VULKAN_1_2);
```

Comment:
Nit: Use `bindStateRTShaderText` instead. This change is also needed at other spots, I am going to do it.
```suggestion
    VkShaderObj chit_shader(this, bindStateRTShaderText, VK_SHADER_STAGE_CLOSEST_HIT_BIT_KHR, SPV_ENV_VULKAN_1_2);
```

Reply:
Done